lib/mayaHydra/hydraExtensions/mayaUtils.cpp

```diff
@@ -158,4 +159,43 @@ bool IsAMayaVisibilityAttribute(const MPlug& plug, bool& outVal)
     return isVisibility;
 }
 
+MObject GetShadingGroupFromShader(const MObject& shader)
+{
+    MObject           shadingGroup;
+    MFnDependencyNode fn(shader);
+
+    // Get the "outColor" plug of the shader
+    MPlug outColorPlug = fn.findPlug("outColor", true);
+
+    // Get the connected plugs
+    MPlugArray connectedPlugs;
+    outColorPlug.connectedTo(connectedPlugs, false, true);
+
+    // Loop over the connected plugs
+    for (unsigned int i = 0; i < connectedPlugs.length(); ++i) {
+        MObject node = connectedPlugs[i].node();
+        if (node.apiType() == MFn::kShadingEngine) // Check if the node is a shading group
+        {
+            shadingGroup = node;
+            break;
+        }
+    }
+
+    return shadingGroup;
+}
+
+MObject GetDefaultStandardSurfaceShader()
+{
+    MSelectionList list;
+    MObject        node;
+
+    // Add the shader to the selection list
+    list.add("standardSurface1");
+
+    // Get the shader as an MObject
+    list.getDependNode(0, node);
```

Comment:
Nit-pick: isn't this just `GetDependNodeFromNodeName("standardSurface1", node)`?

Reply:
Ha sorry, I missed that function, I will use it now.